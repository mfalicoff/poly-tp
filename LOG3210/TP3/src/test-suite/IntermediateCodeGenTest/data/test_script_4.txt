Declaration
num a;
num b;
num c;
EndDeclaration

script1 {
  a = 1;
}

script2 {
  c = 3;
}

script3 {
  b = 2;
}

script3()
script1()
