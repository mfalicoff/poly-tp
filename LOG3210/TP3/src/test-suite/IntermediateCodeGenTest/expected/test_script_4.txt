b = 2
a = 1
_L0
